### DISCUSSION TITLE
Get Events (by account) but on a contract

### DISCUSSION THREAD

**Discussion by npmTUser** [ProjectOpenSea/opensea-js#1355] [General]
Hi,
Is it possible to retrieve a wallet's transactions on a given contract? Currently we have Get Events (by account) but you cannot filter by contract

**Comment by ryanio** [comment]
sorry not currently, you'll just have to parse Get Events (by account)

**Reply by npmTUser** [reply]
Understood, do you plan to add this endpoint in the future?
